### PR TITLE
[FEATURE] Adding instructions plugin CLCAD-68

### PR DESCRIPTION
**Pull request by harlie**
This pr creates a plugin that adds an instructions page and a line of instructions to each collection type. It also allows admins to customize the content via Settings -> Instructions Content. 

image

### CODE REVIEW

**Comment by jcoryalvernaz** [comment]
@harlie Thanks for putting this together! I think this approach does what we need.

**Comment by harlie** [comment]
@jcoryalvernaz - modified the approach to use [<Portal>](https://design-system-git-main-strapijs.vercel.app/?path=/docs/design-system-technical-components-portal--docs) instead. Added an instructions homepage and a settings page to allow for admin configuration. I think it's ready enough for a real review now

**Review by jcoryalvernaz — CHANGES_REQUESTED** [review]
Code LGTM. I have comments related to maintenance. Thanks!

File: backend/src/plugins/instructions/admin/src/components/Injected/list-view-instructions.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const ListViewInstructions = () => {
+    const headerElement = document.querySelector('[data-strapi-header="true"]');
```
[the remark this answers is not in the document]
Comment (harlie) [reply backend/src/plugins/instructions/admin/src/components/Injected/list-view-instructions.tsx]: @jcoryalvernaz unfortunately there is no list view equivalent to useCMEditViewDataManager that I could find. Folks on the internet recommended diving into strapi's state to pull out data which I do in [this line here in a later commit](https://github.com/maplight/clc-ad-transparency/pull/30/commits/a473f8608f13104caeb0afec21e615c00ad64b16#diff-59963eb93ab42070147f2a6e378c78b34ba1c9345ef15709d7aa9c7126e0be6eL13:~:text=const%20apiID%20%3D%20useSelector((state%20%3A%20any)%20%3D%3E%20state%5B%22content%2Dmanager_listView%22%5D.contentType.apiID)%3B):

>  const apiID = useSelector((state : any) => state["content-manager_listView"].contentType.apiID);
I then use the apiID to fetch the appropriate content for that list page.

File: backend/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "develop": "strapi develop",
     "start": "strapi start",
     "build": "strapi build && yarn build:plugin",
-    "build:plugin": "cd ./src/plugins/ad-disclosure-report && yarn build",
+    "build:plugin": "cd ./src/plugins/ad-disclosure-report && yarn build && cd ../../../src/plugins/instructions && yarn build",
```
[the remark this answers is not in the document]
Comment (harlie) [reply backend/package.json:9]: Probably not a bad idea. I can take a stab at it this week, mostly concerned about any migration needs.

**Comment by harlie** [comment]
closing in favor of #44

**Pull request closed without merging**